### PR TITLE
Fix docs (type annotations and docstrings)

### PR DESCRIPTION
**Pull request by alessiamarcolini**

### CODE REVIEW

File: src/histolab/filters/image_filters_functional.py [hunk 1]
```diff
@@ -21,60 +21,61 @@
 from functools import reduce
 
 import numpy as np
+import PIL
+import PIL.ImageOps
```
Comment (nicolebussola) [inline src/histolab/filters/image_filters_functional.py:25]: Is it necessary to import also PIL.ImageOps?

File: src/histolab/util.py [hunk 2]
```diff
@@ -23,15 +23,16 @@
 from typing import Tuple
 
 import numpy as np
+import PIL
+import PIL.ImageDraw
```
Comment (nicolebussola) [inline src/histolab/util.py:27]: same question here

File: src/histolab/filters/image_filters_functional.py (hunk 1 above)
Comment (alessiamarcolini) [reply src/histolab/filters/image_filters_functional.py:25]: it seems necessary. Importing only PIL and calling `PIL.ImageOps.function` seems not possible. @ernestoarbitrio any clues?

Comment (alessiamarcolini) [reply src/histolab/filters/image_filters_functional.py:25]: Offline he said he has no clues. I'll leave as it is for the moment

**Pull request merged**